include/field2d.hxx

```diff
@@ -149,79 +149,79 @@ class Field2D : public Field, public FieldData {
   }
-  inline const BoutReal& operator()(int jx, int jy) const {
+  inline const BoutReal &operator()(int jx, int jy) const {
 #if CHECK > 2
-    if(!isAllocated())
+    if (!isAllocated())
       throw BoutException("Field2D: () operator on empty data");
-    
-    if((jx < 0) || (jx >= nx) || 
-       (jy < 0) || (jy >= ny) )
-      throw BoutException("Field2D: (%d, %d) index out of bounds (%d , %d)\n", 
-                          jx, jy, nx, ny);
+
+    if ((jx < 0) || (jx >= nx) || (jy < 0) || (jy >= ny))
+      throw BoutException("Field2D: (%d, %d) index out of bounds (%d , %d)\n", jx, jy, nx,
+                          ny);
 #endif
-  
-    return data[jx*ny + jy];
+
+    return data[jx * ny + jy];
   }
 
   /*!
    * DIrect access to underlying array. This version is for compatibility
    * with Field3D objects
    */
-  BoutReal& operator()(int jx, int jy, int UNUSED(jz)) {
-    return operator()(jx, jy);
-  }
-  const BoutReal& operator()(int jx, int jy, int UNUSED(jz)) const {
+  BoutReal &operator()(int jx, int jy, int UNUSED(jz)) { return operator()(jx, jy); }
+  const BoutReal &operator()(int jx, int jy, int UNUSED(jz)) const {
     return operator()(jx, jy);
   }
-  
-  Field2D & operator+=(const Field2D &rhs); ///< In-place addition. Copy-on-write used if data is shared
-  Field2D & operator+=(BoutReal rhs);       ///< In-place addition. Copy-on-write used if data is shared
-  Field2D & operator-=(const Field2D &rhs); ///< In-place subtraction. Copy-on-write used if data is shared
-  Field2D & operator-=(BoutReal rhs);       ///< In-place subtraction. Copy-on-write used if data is shared
-  Field2D & operator*=(const Field2D &rhs); ///< In-place multiplication. Copy-on-write used if data is shared
-  Field2D & operator*=(BoutReal rhs);       ///< In-place multiplication. Copy-on-write used if data is shared
-  Field2D & operator/=(const Field2D &rhs); ///< In-place division. Copy-on-write used if data is shared
-  Field2D & operator/=(BoutReal rhs);       ///< In-place division. Copy-on-write used if data is shared
+
+  Field2D &operator+=(
+      const Field2D &rhs); ///< In-place addition. Copy-on-write used if data is shared
+  Field2D &
+  operator+=(BoutReal rhs); ///< In-place addition. Copy-on-write used if data is shared
+  Field2D &operator-=(
+      const Field2D &rhs); ///< In-place subtraction. Copy-on-write used if data is shared
+  Field2D &operator-=(
+      BoutReal rhs); ///< In-place subtraction. Copy-on-write used if data is shared
+  Field2D &operator*=(const Field2D &rhs); ///< In-place multiplication. Copy-on-write
+                                           ///used if data is shared
+  Field2D &operator*=(
+      BoutReal rhs); ///< In-place multiplication. Copy-on-write used if data is shared
+  Field2D &operator/=(
+      const Field2D &rhs); ///< In-place division. Copy-on-write used if data is shared
+  Field2D &
+  operator/=(BoutReal rhs); ///< In-place division. Copy-on-write used if data is shared
```

Comment:
clang... sigh :(

Reply:
This could be fixed by moving the comment before the function ...

Probably will have to try a different approach to merge ...